source/examples/generated/dotnet/AsynchronousTransactions.snippet.write-async.cs

```diff
@@ -0,0 +1,11 @@
+var person = await realm.WriteAsync(() =>
+{
+    return realm.Add(new Person()
+    {
+        Name = "John Doe"
+    });
+});
```

Comment:
Can be simplified like so
```suggestion
var person = await realm.WriteAsync(() =>
    realm.Add(new Person()
    {
        Name = "John Doe"
    })
);
```